pilot/pkg/model/destination_rule.go

```diff
@@ -120,3 +127,15 @@ func (ps *PushContext) inheritDestinationRule(parent, child *config.Config) *con
 
 	return &merged
 }
+
+func workloadSelectorEquals(a, b map[string]string) bool {
```

Comment:
looks like this already exists https://github.com/istio/istio/blob/35b6de45318e1f52544fd747f532732aac639f5d/pkg/config/labels/instance.go#L76

Reply:
+1:

```
labels.Instance(a).Equals(b)
```

Reply:
Changed